frontend/components/Card.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="card">
+        {{ content }}
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'Card',
+        props: {
+            content: String
+        },
+        data () {
+            return {}
+        },
+    }
```

Comment:
I think we should remove one level of indentation so that `<script>` contents start unindented, to match `<style>` indentation. Thoughts?